sql/sql_connect.cc

```diff
@@ -655,6 +649,49 @@ void update_global_user_stats(THD* thd, bool create_user, time_t now)
   mysql_mutex_unlock(&LOCK_global_user_client_stats);
 }
 
+static void clear_stats_concurrent_connections(HASH* stats) {
+  for (ulong idx=0; idx < stats->records; idx++)
+  {
+    USER_STATS* user_stats=
+      reinterpret_cast<USER_STATS*>(my_hash_element(stats, idx));
+    user_stats->concurrent_connections = 0;
+  }
+}
+
+static void inc_stats_concurrent_conn(HASH* stats, const char* user_string, int cnt) {
+  USER_STATS* user_stats;
+  if ((user_stats = (USER_STATS *) my_hash_search(stats,
```

Comment:
Move assignment to var declaration (can make the pointer itself const too)
C++ style cast